zookeeper/core/partial_component_test.py

```diff
@@ -1,9 +1,11 @@
 import pytest
 
 from zookeeper.core.component import component, configure
-from zookeeper.core.field import ComponentField, Field
+from zookeeper.core.field import ComponentField, Field  # type: ignore
 from zookeeper.core.partial_component import PartialComponent
 
+# pytype: disable=invalid-annotation
```

Comment:
Should just disable type checking for `*_test.py` instead?

Reply:
Yeah, thinking about it it's probably easier to just do that.